bin/run_epix

```diff
@@ -1,122 +1,133 @@
+    parser.add_argument('--EntryStop', dest='EntryStop', type=int,
+                        action='store',
+                        help='Number of entries to read from first. Defaulted to all')
+    parser.add_argument('--MicroSeparation', dest='MicroSeparation', type=float,
+                        action='store', default=0.05,
+                        help='Spatial resolution for DBSCAN micro-clustering [mm]')
+    parser.add_argument('--MicroSeparationTime', dest='MicroSeparationTime', type=float,
+                        action='store', default=10,
+                        help='Time resolution for DBSCAN micro-clustering [ns]')
+    parser.add_argument('--TagClusterBy', dest='TagClusterBy', type=str,
+                        action='store', default='time',
+                        help=('Classification of the type of particle of a cluster, '
+                              'based on most energetic contributor ("energy") or first '
+                              'depositing particle ("time")'),
+                        choices={'time', 'energy'})
+    parser.add_argument('--MaxDelay', dest='MaxDelay', type=float,
+                        action='store', default=1e7, #ns
+                        help='Maximal time delay to first interaction which will be stored [ns]')
+    parser.add_argument('--OutputPath', dest='OutputPath',
+                       action='store', default="",
+                       help=('Optional output path. If not specified the result will be saved'
+                             'in the same dir as the input file.'))
+    parser.add_argument('--Debug', dest='debug',
+                       action='store_true', default="",
+                       help=('If specifed additional information is printed to the consol.')
+                        )
+
+    args = parser.parse_args(sys.argv[1:])
+    return args
+
+
+def main(args, return_df=False):
+    # TODO: remove setup from main for strax
+    path, file, detector, outer_cylinder = setup(args)
+
+    if args.debug:
+        print("epix configuration: ", args)
+        # TODO: also add memory information see starxer and change this to debug
+        # Getting time information:
         starttime = time.time()
         tnow = starttime
-    # Loading the data:
-    p = args.InputFile
-    p = p.split('/')
-    if p[0] == "":
-        p[0] = "/"
-    path = os.path.join(*p[:-1])
-    file_name = p[-1]
-    print(f'Reading in root file {file_name}')
-    inter = epix.loader(path, file_name, kwargs_uproot_ararys={'entry_stop': args.EntryStop})
-    if args.Timing:
-        t = time.time()
-        print(f'It took {round(t - tnow, 5)} sec to load the data.')
-        tnow = t
-
-    print((f'Finding clusters of interactions with a dr = {args.MicroSeparation} mm'
-           f' and dt = {args.MicroSeparationTime} ns'))
-    inter = epix.find_cluster(inter, args.MicroSeparation/10, args.MicroSeparationTime)
```

Comment:
Should not we just change the corresponding option to cm?

Reply:
I find that, for this particular input, units in mm or microns are more intuitive for the user.